### DISCUSSION TITLE
Problème installation / Installation problem

### DISCUSSION THREAD

**Discussion by Astraa007** [dbarzin/deming#182] [Q&A]
Hello,

First of all, thank you very much for this tool, which should eventually relieve many CISOs. I have encountered a problem with the installation. When I want to access the “Controls” tab, I can't do it. I have this error (see attachments). Has anyone encountered this problem before? If so, how can I solve it?
Many thanks for your help.

Best regards,

______________________________________________________________________________________________________________________________________________

Bonjour,

Tout d'abord un grand merci pour cet outil qui à terme devrait soulager beaucoup de RSSI. J'ai rencontré un souci au niveau de l'installation. Lorsque je veux accéder à l'onglet "Mesures" ("Controls" en anglais), je n'y arrive pas. J'ai cette erreur (cf. pièces jointes). Quelqu'un a déjà rencontré ce problème ? Si oui, comment le résoudre ?
Un grand merci pour votre aide.

Cordialement,
![Erreur bouton mesures 1](https://github.com/user-attachments/assets/30af808f-5659-453c-8d55-d28993d15a0c)
![Erreur bouton mesures](https://github.com/user-attachments/assets/d5254077-59d5-495c-bec0-259b0b2e3976)

**Comment by dbarzin** [comment]
Tu utilises quelle base de données ? Peux-tu installer la dernière version de l'application ?

**Reply by Astraa007** [reply]
Bonjour, Merci pour le retour.
J'utilise postgresql. Je ne pense pas que ce soit la dernière version car sauf erreur de ma part, je vois qu'une dernière release a été faite il y'a 2 semaines.

**Reply by dbarzin** [reply]
Ce sera corrigé pour lundi.

**Comment by dbarzin** [comment]
OK, corrigé : https://github.com/dbarzin/deming/pull/196/files

**Reply by Astraa007** [reply]
Bonjour, je confirme que ça marche. Un grand merci !
